source/common/upstream/cluster_manager_impl.h

```diff
@@ -151,6 +151,7 @@ class ClusterManagerInitHelper : Logger::Loggable<Logger::Id::upstream> {
   COUNTER(cluster_updated_via_merge)                                                               \
   COUNTER(update_merge_cancelled)                                                                  \
   COUNTER(update_out_of_merge_window)                                                              \
+  COUNTER(upstream_connections_closed_on_host_set_change)                                          \
```

Comment:
In the interest of not wasting memory on a stat unless we really need it, do you think this stat is useful? I feel like the user will know if the option is configured or not and can look at connection close stats? WDYT?

Reply:
@mattklein123 was not aware that connection close stats is available. Agree, we could get rid of it.